test/playground.html

```diff
@@ -71,7 +71,7 @@
 
       autocomplete('#contacts2', {
         debug: true,
-        keyboardShortcuts: [191, 83],
+        keyboardShortcuts: [191, 's'],
```

Comment:
Is there a use case where you would want to specify keycodes instead of simple letters? I feel we could just allow letters for now and wait for keyCodes to be a real issue/differentiator